.moon/tasks/go.yml

```diff
@@ -37,6 +37,8 @@ tasks:
 
   start:
     command: "$workspaceRoot/bin/$project"
+    deps:
```

Comment:
I split this up on purpose because I wanted them to be separate steps in the dockerfile for the integration test. Agree it's not ideal but shouldn't really get in the way for dev, right?